modules/cnv-edit-boot-order-yaml-web.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-edit-boot-order-yaml-web_{context}"]
+= Editing a boot order list in the YAML configuration file.
+
+Edit the boot order list list in a YAML configuration file by using the CLI.
+
+.Prerequisites
+. You must have access to the OpenShift Container Platform cluster.
+. You must have installed the CLI.
+
+.Procedure
+. Open the YAML configuration file for the virtual machine by running the following command.
+
+----
+$ oc edit vm example
+----
+. Edit the YAML file and modify the values for the boot order associated with a disk or Network Interface Card (NIC).
+
+. For example:
++
+[source,yaml]
+----
+disks:
```

Comment:
I think that we need `-` in the begining of each disk and interface item ?
see example:
```
          disks:
            - bootOrder: 1
              disk:
                bus: virtio
              name: containerdisk
            - disk:
                bus: virtio
              name: cloudinitdisk
            - cdrom:
                bus: virtio
              name: cd-drive-1
          interfaces:
            - bootOrder: 2
              macAddress: '02:96:c4:00:00:00'
              masquerade: {}
              name: default
```

Reply:
you forgot the `<1>` and `<2>` after the `bootOrder` items in the example yaml :-)